### PR TITLE
Added completed_bps metric to existing guidestar metrics

### PR DESCRIPTION
**Pull request by bspidel**
For each aligned bible, new code opens the language zip file and counts the number of books that contain any "\zaln-s" tags indicating start of alignment. It then verifies that there are all of tA, tN, tQ, and tW resources in the catalog

Number of completed_bps detected at build time was 132

To Test:
Verify that docker container is automatically deployed.
Verify that dashboard displayed number for completed_bps is near what was generated during build.

### CODE REVIEW

File: gatherer.py [hunk 1]
```diff
@@ -99,6 +123,41 @@ def catalog(gl_codes, metrics={}):
                 metrics['subject_{}'.format(y['subject'])] =0
             metrics['subject_{}'.format(y['subject'])] +=1
     metrics['all_resources'] = all_resources
+    isTa = False
+    isTn = False
+    isTq = False
+    isTw = False
```
Comment (jag3773) [inline gatherer.py]: @bspidel Why not set these under line 131 *within* the language loop?  That way you don't have to reset them separately in 156-159.  That is, if I'm reading this correctly...

Comment (bspidel) [reply gatherer.py]: You are right. I will change.

Comment (bspidel) [reply gatherer.py]: Changes moving resource flag mgmt into language loop, commited.

Comment (bspidel) [reply gatherer.py]: After testing I noted that there was an error in preexisting code that I presume was related to github credentials.

**Pull request merged**